guides/common/assembly_api-requests-in-various-languages.adoc

```diff
@@ -1,7 +1,29 @@
 include::modules/con_api-requests-in-various-languages.adoc[]
 
-include::modules/proc_calling-the-api-in-curl.adoc[leveloffset=+1]
+include::modules/con_calling-the-api-in-curl.adoc[leveloffset=+1]
 
-include::modules/proc_calling-the-api-in-ruby.adoc[leveloffset=+1]
+include::modules/proc_passing-json-data-to-the-api-request.adoc[leveloffset=+2]
 
-include::modules/proc_calling-the-api-in-python.adoc[leveloffset=+1]
+include::modules/con_retrieving-a-list-of-resources.adoc[leveloffset=+2]
+
+include::modules/proc_listing-users.adoc[leveloffset=+3]
+
+include::modules/proc_creating-and-modifying-resources.adoc[leveloffset=+2]
+
+// TODO: Only include parts?
+//       Use tags to mitigate this?
+include::modules/proc_creating-a-user.adoc[leveloffset=+2]
```

Comment:
:point_up:

Reply:
I'm not sure what the comments are referring to. Why do you suggest to include only parts? And mitigate what exactly?

Reply:
Yes, much simpler. -> Implemented.

For now, the API guide hide web and CLI procedures; the admin guide shows the API procedures too.